lib/engine/game/g_18_eus/game.rb

```diff
@@ -469,8 +520,42 @@ def company_status_str(company)
           return "Bid box #{index + 1}" if index && index < self.class::BIDDING_BOX_PRIVATE_COUNT
         end
 
-        def bidding_token_per_player
-          self.class::BIDDING_BOX_PRIVATE_COUNT
+        def revenue_for(route, stops)
```

Comment:
Will need to merge my version of this in

Reply:
I'll handle merging the two.

Should bidding_token_per_player still be there? Or did I move it elsewhere?